src/containers/mobile/nrc-landing/country-card/country-card-component.jsx

```diff
@@ -0,0 +1,169 @@
+import React, { useEffect, useState, useMemo } from 'react';
+
+import { loadModules } from 'esri-loader';
+
+import { T, useT, useLocale } from '@transifex/react';
+
+import cx from 'classnames';
+import { motion } from 'framer-motion';
+
+import { LAND_MARINE } from 'constants/country-mode-constants';
+import { getCountryNames } from 'constants/translation-constants';
+
+import { ReactComponent as CloseIcon } from 'icons/close.svg';
+
+import styles from './country-card-styles.module.scss';
+
+function CountryEntryTooltipComponent({
```

Comment:
Is it possible to reuse the existing NRC tooltip? I feel this could be too much to maintain